heroes.js

```diff
@@ -20,7 +20,7 @@ function createHeroBlock(hero) {
 
 function setupHeroReveal() {
   for (var i = 0; i < heroes.length; i++) {
-    var hero = heroes[i]
+    const hero = heroes[i]
```

Comment:
`const` totally works here. Since it is block scoped each iteration of the for loop will have its own scope and therefore each version of the mouseover will have a different parent scope, one which has the correct hero in it.

The other option would be to use a `forEach` like the setupHeroBlocks function since the inner function of the `forEach` would also have its own scope.